e2e/test/provisioning/ProvisioningServiceClientE2ETests.cs

```diff
@@ -327,13 +341,18 @@ public async Task ProvisioningServiceClient_GroupEnrollments_Create_Ok(string pr
             ReprovisionPolicy reprovisionPolicy,
             AllocationPolicy allocationPolicy,
             CustomAllocationDefinition customAllocationDefinition,
-            ICollection<string> iotHubsToProvisionTo,
+            IList<string> iotHubsToProvisionTo,
             Devices.Provisioning.Service.DeviceCapabilities capabilities,
             MsTestLogger logger)
         {
             Attestation attestation;
             IndividualEnrollment individualEnrollment;
             IndividualEnrollment createdEnrollment = null;
+            if (iotHubsToProvisionTo == null)
```

Comment:
We can't assign the whole list over anymore, because of the private setter. In order to guard the foreach done later, I just make this an empty list.